src/main/java/com/kuzmins/dao/UserDAO.java

```diff
@@ -0,0 +1,68 @@
+package com.kuzmins.dao;
+
+import com.kuzmins.config.ConnectionConfig;
+import com.kuzmins.model.User;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+public class UserDAO {
+
+    private final static Connection CONNECTION = ConnectionConfig.getConnection();
```

Comment:
open (and close) connection and resources for each method, do not use one opened connection for class level.
You can do it:
```java
        try {
            Connection conn= ConnectionConfig.getConnection();
            PreparedStatement preparedStatement =conn.prepareStatement(INSERT_USER_QUERY);
            ...
        } catch (SQLException e) {
           ...
        } finally {
            try {
                preparedStatement.close();
                conn.close();
            } catch (SQLException e) {
                throw new RuntimeException(e);
            }
        }
```
or in `try-with-resources` construction:
```java
  try (Connection connection = ConnectionConfig.getConnection();
             PreparedStatement statement = connection.prepareStatement(sqlCommand)) {
             ...
        } catch (SQLException e) {
            System.out.println(e);
        } 
```

Reply:
applied try-with-resources construction